com.hazelcast.jet.sql.SqlErrorTest.testUserCancel

`ramizdundar:configuration_reload` (commit https://github.com/hazelcast/hazelcast/pull/20260/commits/3b9ac4ae2f3b91eeddfc765965b2eaf6cdd297d6)

Failed on `Hazelcast-pr-builder`: https://jenkins.hazelcast.com/job/Hazelcast-pr-builder/9076/testReport/junit/com.hazelcast.jet.sql/SqlErrorTest/testUserCancel/

<details><summary>Stacktrace:</summary>

```
java.lang.AssertionError: Exception is not thrown
	at org.junit.Assert.fail(Assert.java:89)
	at com.hazelcast.jet.sql.SqlErrorAbstractTest.checkUserCancel(SqlErrorAbstractTest.java:141)
	at com.hazelcast.jet.sql.SqlErrorTest.testUserCancel(SqlErrorTest.java:109)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at com.hazelcast.test.FailOnTimeoutStatement$CallableStatement.call(FailOnTimeoutStatement.java:115)
	at com.hazelcast.test.FailOnTimeoutStatement$CallableStatement.call(FailOnTimeoutStatement.java:107)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:748)
```

</details>

<details><summary>Standard output:</summary>

```
Finished Running Test: testParsingError in 0.102 seconds.
Started Running Test: testUserCancel
09:31:18,113  INFO |testUserCancel| - [MetricsConfigHelper] testUserCancel - [LOCAL] [dev] [5.1-SNAPSHOT] Overridden metrics configuration with system property 'hazelcast.metrics.collection.frequency'='1' -> 'MetricsConfig.collectionFrequencySeconds'='1'
09:31:18,115  INFO |testUserCancel| - [logo] testUserCancel - [127.0.0.1]:5701 [dev] [5.1-SNAPSHOT] 
	+       +  o    o     o     o---o o----o o      o---o     o     o----o o--o--o
	+ +   + +  |    |    / \       /  |      |     /         / \    |         |   
	+ + + + +  o----o   o   o     o   o----o |    o         o   o   o----o    |   
	+ +   + +  |    |  /     \   /    |      |     \       /     \       |    |   
	+       +  o    o o       o o---o o----o o----o o---o o       o o----o    o   
09:31:18,115  INFO |testUserCancel| - [system] testUserCancel - [127.0.0.1]:5701 [dev] [5.1-SNAPSHOT] Copyright (c) 2008-2021, Hazelcast, Inc. All Rights Reserved.
09:31:18,115  INFO |testUserCancel| - [system] testUserCancel - [127.0.0.1]:5701 [dev] [5.1-SNAPSHOT] Hazelcast Platform 5.1-SNAPSHOT (20220110 - 5e78d1d) starting at [127.0.0.1]:5701
09:31:18,115  INFO |testUserCancel| - [system] testUserCancel - [127.0.0.1]:5701 [dev] [5.1-SNAPSHOT] Cluster name: dev
09:31:18,123  INFO |testUserCancel| - [MetricsConfigHelper] testUserCancel - [127.0.0.1]:5701 [dev] [5.1-SNAPSHOT] Collecting debug metrics and sending to diagnostics is enabled
09:31:18,132  WARN |testUserCancel| - [CPSubsystem] testUserCancel - [127.0.0.1]:5701 [dev] [5.1-SNAPSHOT] CP Subsystem is not enabled. CP data structures will operate in UNSAFE mode! Please note that UNSAFE mode will not provide strong consistency guarantees.
09:31:18,139  INFO |testUserCancel| - [JetServiceBackend] testUserCancel - [127.0.0.1]:5701 [dev] [5.1-SNAPSHOT] Setting number of cooperative threads and default parallelism to 2
09:31:18,145  INFO |testUserCancel| - [Diagnostics] testUserCancel - [127.0.0.1]:5701 [dev] [5.1-SNAPSHOT] Diagnostics disabled. To enable add -Dhazelcast.diagnostics.enabled=true to the JVM arguments.
09:31:18,146  INFO |testUserCancel| - [LifecycleService] testUserCancel - [127.0.0.1]:5701 [dev] [5.1-SNAPSHOT] [127.0.0.1]:5701 is STARTING
09:31:18,146  INFO |testUserCancel| - [JetExtension] testUserCancel - [127.0.0.1]:5701 [dev] [5.1-SNAPSHOT] Jet is enabled
09:31:18,146  INFO |testUserCancel| - [ClusterService] testUserCancel - [127.0.0.1]:5701 [dev] [5.1-SNAPSHOT] 

Members {size:1, ver:1} [
	Member [127.0.0.1]:5701 - beef212d-c629-49e3-908b-032fd4b75b0d this
]

09:31:18,146 DEBUG |testUserCancel| - [JobCoordinationService] hz.distracted_wilbur.cached.thread-1 - [127.0.0.1]:5701 [dev] [5.1-SNAPSHOT] Not starting jobs because partitions are not yet initialized.
09:31:18,146  INFO |testUserCancel| - [LifecycleService] testUserCancel - [127.0.0.1]:5701 [dev] [5.1-SNAPSHOT] [127.0.0.1]:5701 is STARTED
09:31:18,147  INFO |testUserCancel| - [PartitionStateManager] testUserCancel - [127.0.0.1]:5701 [dev] [5.1-SNAPSHOT] Initializing cluster partition table arrangement...
09:31:18,150  INFO |testUserCancel| - [HealthMonitor] hz.distracted_wilbur.HealthMonitor - [127.0.0.1]:5701 [dev] [5.1-SNAPSHOT] processors=16, physical.memory.total=377.6G, physical.memory.free=86.5G, swap.space.total=4.0G, swap.space.free=4.0G, heap.memory.used=259.5M, heap.memory.free=993.0M, heap.memory.total=1.2G, heap.memory.max=1.8G, heap.memory.used/total=20.67%, heap.memory.used/max=14.22%, minor.gc.count=13, minor.gc.time=871ms, major.gc.count=4, major.gc.time=803ms, load.process=6.25%, load.system=100.00%, load.systemAverage=128.60, thread.count=53, thread.peakCount=278, cluster.timeDiff=0, event.q.size=0, executor.q.async.size=0, executor.q.client.size=0, executor.q.client.query.size=0, executor.q.client.blocking.size=0, executor.q.query.size=0, executor.q.scheduled.size=0, executor.q.io.size=0, executor.q.system.size=0, executor.q.operations.size=0, executor.q.priorityOperation.size=0, operations.completed.count=0, executor.q.mapLoad.size=0, executor.q.mapLoadAllKeys.size=0, executor.q.cluster.size=0, executor.q.response.size=0, operations.running.count=0, operations.pending.invocations.percentage=0.00%, operations.pending.invocations.count=0, proxy.count=1, clientEndpoint.count=0, connection.active.count=0, client.connection.count=0, connection.count=0
09:31:18,158  INFO |testUserCancel| - [ClientInvocationService] testUserCancel - hz.client_6 [dev] [5.1-SNAPSHOT] Running with 2 response threads, dynamic=true
09:31:18,159  INFO |testUserCancel| - [LifecycleService] testUserCancel - hz.client_6 [dev] [5.1-SNAPSHOT] HazelcastClient 5.1-SNAPSHOT (20220110 - 5e78d1d) is STARTING
09:31:18,159  INFO |testUserCancel| - [LifecycleService] testUserCancel - hz.client_6 [dev] [5.1-SNAPSHOT] HazelcastClient 5.1-SNAPSHOT (20220110 - 5e78d1d) is STARTED
09:31:18,160  INFO |testUserCancel| - [ClientConnectionManager] testUserCancel - hz.client_6 [dev] [5.1-SNAPSHOT] Trying to connect to cluster: dev
09:31:18,160  INFO |testUserCancel| - [ClientConnectionManager] testUserCancel - hz.client_6 [dev] [5.1-SNAPSHOT] Trying to connect to [127.0.0.1]:5701
09:31:18,160  INFO |testUserCancel| - [HazelcastClient] testUserCancel - Created connection to endpoint: [127.0.0.1]:5701, connection: MockedClientConnection{localAddress=[127.0.0.1]:40001, super=ClientConnection{alive=true, connectionId=1, channel=null, remoteAddress=null, lastReadTime=never, lastWriteTime=never, closedTime=never, connected server version=null}}
09:31:18,161 DEBUG |testUserCancel| - [AuthenticationMessageTask] hz.distracted_wilbur.generic-operation.thread-0 - [127.0.0.1]:5701 [dev] [5.1-SNAPSHOT] Processing authentication with clientUuid b01d7491-3bc3-4517-9713-c77e98d4c0c5 and clientName hz.client_6
09:31:18,161  INFO |testUserCancel| - [AuthenticationMessageTask] hz.distracted_wilbur.generic-operation.thread-0 - [127.0.0.1]:5701 [dev] [5.1-SNAPSHOT] Received auth from MockedNodeConnection{ remoteAddress = [127.0.0.1]:40001, localAddress = [127.0.0.1]:5701, connectionId = 1}, successfully authenticated, clientUuid: b01d7491-3bc3-4517-9713-c77e98d4c0c5, client name: hz.client_6, client version: 5.1-SNAPSHOT
09:31:18,161  INFO |testUserCancel| - [LifecycleService] testUserCancel - hz.client_6 [dev] [5.1-SNAPSHOT] HazelcastClient 5.1-SNAPSHOT (20220110 - 5e78d1d) is CLIENT_CONNECTED
09:31:18,161  INFO |testUserCancel| - [ClientConnectionManager] testUserCancel - hz.client_6 [dev] [5.1-SNAPSHOT] Authenticated with server [127.0.0.1]:5701:beef212d-c629-49e3-908b-032fd4b75b0d, server version: 5.1-SNAPSHOT, local address: /127.0.0.1:40001
09:31:18,162  INFO |testUserCancel| - [Diagnostics] testUserCancel - hz.client_6 [dev] [5.1-SNAPSHOT] Diagnostics disabled. To enable add -Dhazelcast.diagnostics.enabled=true to the JVM arguments.
09:31:18,162 TRACE |testUserCancel| - [ClientListenerService] hz.client_6.internal-1 - hz.client_6 [dev] [5.1-SNAPSHOT] Register attempt of ClusterViewListenerHandler to MockedClientConnection{localAddress=[127.0.0.1]:40001, super=ClientConnection{alive=true, connectionId=1, channel=null, remoteAddress=[127.0.0.1]:5701, lastReadTime=2022-01-10 09:31:18.161, lastWriteTime=2022-01-10 09:31:18.160, closedTime=never, connected server version=5.1-SNAPSHOT}}
09:31:18,163  INFO |testUserCancel| - [ClientClusterService] hz.client_6.event-95 - hz.client_6 [dev] [5.1-SNAPSHOT] 

Members [1] {
	Member [127.0.0.1]:5701 - beef212d-c629-49e3-908b-032fd4b75b0d
}

09:31:18,165 TRACE |testUserCancel| - [ClientListenerService] hz.client_6.eventRegistration- - hz.client_6 [dev] [5.1-SNAPSHOT] Register attempt of ClientListenerRegistration{codec=com.hazelcast.client.impl.spi.impl.ClientInvocationServiceImpl$1@3d450783, handler=com.hazelcast.client.impl.spi.impl.ClientInvocationServiceImpl$BackupEventHandler@99f7e6b} to MockedClientConnection{localAddress=[127.0.0.1]:40001, super=ClientConnection{alive=true, connectionId=1, channel=null, remoteAddress=[127.0.0.1]:5701, lastReadTime=2022-01-10 09:31:18.164, lastWriteTime=2022-01-10 09:31:18.162, closedTime=never, connected server version=5.1-SNAPSHOT}}
09:31:18,166 TRACE |testUserCancel| - [ClientListenerService] hz.client_6.eventRegistration- - hz.client_6 [dev] [5.1-SNAPSHOT] Registered ClientListenerRegistration{codec=com.hazelcast.client.impl.spi.impl.ClientInvocationServiceImpl$1@3d450783, handler=com.hazelcast.client.impl.spi.impl.ClientInvocationServiceImpl$BackupEventHandler@99f7e6b} to MockedClientConnection{localAddress=[127.0.0.1]:40001, super=ClientConnection{alive=true, connectionId=1, channel=null, remoteAddress=[127.0.0.1]:5701, lastReadTime=2022-01-10 09:31:18.166, lastWriteTime=2022-01-10 09:31:18.166, closedTime=never, connected server version=5.1-SNAPSHOT}}
09:31:18,167  INFO |testUserCancel| - [ClientStatisticsService] testUserCancel - Client statistics is enabled with period 5 seconds.
09:31:18,173 TRACE |testUserCancel| - [ClientListenerService] ForkJoinPool.commonPool-worker-15 - hz.client_6 [dev] [5.1-SNAPSHOT] Registered ClusterViewListenerHandler to MockedClientConnection{localAddress=[127.0.0.1]:40001, super=ClientConnection{alive=true, connectionId=1, channel=null, remoteAddress=[127.0.0.1]:5701, lastReadTime=2022-01-10 09:31:18.166, lastWriteTime=2022-01-10 09:31:18.166, closedTime=never, connected server version=5.1-SNAPSHOT}}
09:31:18,173 DEBUG |testUserCancel| - [CalciteSqlOptimizer] testUserCancel - [127.0.0.1]:5701 [dev] [5.1-SNAPSHOT] Before logical opt:
LogicalProject(__key=[$0], this=[$1])
  LogicalTableScan(table=[[hazelcast, public, map[projects=[0, 1]]]])

09:31:18,174 DEBUG |testUserCancel| - [CalciteSqlOptimizer] testUserCancel - [127.0.0.1]:5701 [dev] [5.1-SNAPSHOT] After logical opt:
FullScanLogicalRel(table=[[hazelcast, public, map[projects=[0, 1]]]])

09:31:18,175 DEBUG |testUserCancel| - [CalciteSqlOptimizer] testUserCancel - [127.0.0.1]:5701 [dev] [5.1-SNAPSHOT] After physical opt:
FullScanPhysicalRel(table=[[hazelcast, public, map[projects=[0, 1]]]])

09:31:18,176 DEBUG |testUserCancel| - [LightMasterContext] testUserCancel - [127.0.0.1]:5701 [dev] [5.1-SNAPSHOT] Start executing light job 0765-13d4-c7c0-0001, execution graph in DOT format:
digraph DAG {
	"IMap[public.map]" [localParallelism=1];
	"Project(IMap[public.map])" [localParallelism=default];
	"ClientSink" [localParallelism=1];
	"IMap[public.map]" -> "Project(IMap[public.map])" [label="isolated", queueSize=1024];
	"Project(IMap[public.map])" -> "ClientSink" [label="distributed to [127.0.0.1]:5701-partitioned", queueSize=1024];
}
HINT: You can use graphviz or http://viz-js.com to visualize the printed graph.
09:31:18,176 DEBUG |testUserCancel| - [LightMasterContext] testUserCancel - [127.0.0.1]:5701 [dev] [5.1-SNAPSHOT] Building execution plan for 0765-13d4-c7c0-0001
09:31:18,176 DEBUG |testUserCancel| - [LightMasterContext] testUserCancel - [127.0.0.1]:5701 [dev] [5.1-SNAPSHOT] Built execution plans for 0765-13d4-c7c0-0001
09:31:18,177 DEBUG |testUserCancel| - [InitExecutionOperation] testUserCancel - [127.0.0.1]:5701 [dev] [5.1-SNAPSHOT] Initializing execution plan for job 0765-13d4-c7c0-0001, execution 0765-13d4-c7c0-0001 from [127.0.0.1]:5701
09:31:18,178 DEBUG |testUserCancel| - [JobExecutionService] testUserCancel - [127.0.0.1]:5701 [dev] [5.1-SNAPSHOT] Execution plan for light job ID=0765-13d4-c7c0-0001, jobName='0765-13d4-c7c0-0001', executionId=0765-13d4-c7c0-0001 initialized, will start the execution
09:31:18,184 DEBUG || - [JobExecutionService] ForkJoinPool.commonPool-worker-15 - [127.0.0.1]:5701 [dev] [5.1-SNAPSHOT] Completed execution of job '0765-13d4-c7c0-0001', execution 0765-13d4-c7c0-0001
09:31:18,184 DEBUG || - [JobExecutionService] ForkJoinPool.commonPool-worker-15 - [127.0.0.1]:5701 [dev] [5.1-SNAPSHOT] Execution of job '0765-13d4-c7c0-0001', execution 0765-13d4-c7c0-0001 completed
09:31:18,300 DEBUG |testUserCancel| - [JobRepository] hz.distracted_wilbur.cached.thread-5 - [127.0.0.1]:5701 [dev] [5.1-SNAPSHOT] Job cleanup took 0ms
09:31:18,305  INFO |testUserCancel| - [LifecycleService] Time-limited test - hz.client_6 [dev] [5.1-SNAPSHOT] HazelcastClient 5.1-SNAPSHOT (20220110 - 5e78d1d) is SHUTTING_DOWN
09:31:18,314  INFO |testUserCancel| - [ClientConnectionManager] Time-limited test - hz.client_6 [dev] [5.1-SNAPSHOT] Removed connection to endpoint: [127.0.0.1]:5701:beef212d-c629-49e3-908b-032fd4b75b0d, connection: MockedClientConnection{localAddress=[127.0.0.1]:40001, super=ClientConnection{alive=false, connectionId=1, channel=null, remoteAddress=[127.0.0.1]:5701, lastReadTime=2022-01-10 09:31:18.300, lastWriteTime=2022-01-10 09:31:18.179, closedTime=2022-01-10 09:31:18.314, connected server version=5.1-SNAPSHOT}}
09:31:18,315  INFO |testUserCancel| - [LifecycleService] Time-limited test - hz.client_6 [dev] [5.1-SNAPSHOT] HazelcastClient 5.1-SNAPSHOT (20220110 - 5e78d1d) is CLIENT_DISCONNECTED
09:31:18,315  WARN |testUserCancel| - [TestClientRegistry$MockedServerConnection] pool-36-thread-1 - Server connection closed: null
09:31:18,315  INFO |testUserCancel| - [MockServer] pool-36-thread-1 - [127.0.0.1]:5701 [dev] [5.1-SNAPSHOT] Removed connection to endpoint: [127.0.0.1]:40001, connection: MockedNodeConnection{ remoteAddress = [127.0.0.1]:40001, localAddress = [127.0.0.1]:5701, connectionId = 1}
09:31:18,315  WARN |testUserCancel| - [TwoWayBlockableExecutor] pool-36-thread-1 - Dropping incoming runnable since other end closed. Server Closed EOF. MockedClientConnection{localAddress=[127.0.0.1]:40001, super=ClientConnection{alive=false, connectionId=1, channel=null, remoteAddress=[127.0.0.1]:5701, lastReadTime=2022-01-10 09:31:18.300, lastWriteTime=2022-01-10 09:31:18.179, closedTime=2022-01-10 09:31:18.314, connected server version=5.1-SNAPSHOT}}
09:31:18,315  INFO |testUserCancel| - [ClientEndpointManager] hz.distracted_wilbur.event-94 - [127.0.0.1]:5701 [dev] [5.1-SNAPSHOT] Destroying ClientEndpoint{connection=MockedNodeConnection{ remoteAddress = [127.0.0.1]:40001, localAddress = [127.0.0.1]:5701, connectionId = 1}, clientUuid=b01d7491-3bc3-4517-9713-c77e98d4c0c5, clientName=hz.client_6, authenticated=true, clientVersion=5.1-SNAPSHOT, creationTime=1641807078160, latest clientAttributes=lastStatisticsCollectionTime=1641807078167,enterprise=false,clientType=JVM,clientVersion=5.1-SNAPSHOT,clusterConnectionTimestamp=1641807078160,clientAddress=127.0.0.1,clientName=hz.client_6,credentials.principal=null,os.committedVirtualMemorySize=24438550528,os.freePhysicalMemorySize=92817317888,os.freeSwapSpaceSize=4262916096,os.maxFileDescriptorCount=120000,os.openFileDescriptorCount=325,os.processCpuTime=204490000000,os.systemLoadAverage=128.6,os.totalPhysicalMemorySize=405449981952,os.totalSwapSpaceSize=4294963200,runtime.availableProcessors=16,runtime.freeMemory=1032849520,runtime.maxMemory=1908932608,runtime.totalMemory=1313341440,runtime.uptime=111183,runtime.usedMemory=280491920, labels=[]}
09:31:18,318  INFO |testUserCancel| - [LifecycleService] Time-limited test - hz.client_6 [dev] [5.1-SNAPSHOT] HazelcastClient 5.1-SNAPSHOT (20220110 - 5e78d1d) is SHUTDOWN
09:31:18,318  INFO |testUserCancel| - [LifecycleService] Time-limited test - [127.0.0.1]:5701 [dev] [5.1-SNAPSHOT] [127.0.0.1]:5701 is SHUTTING_DOWN
09:31:18,318 DEBUG |testUserCancel| - [JobCoordinationService] Time-limited test - [127.0.0.1]:5701 [dev] [5.1-SNAPSHOT] Added a shutting-down member: beef212d-c629-49e3-908b-032fd4b75b0d
09:31:18,319  INFO |testUserCancel| - [MigrationManager] hz.distracted_wilbur.cached.thread-1 - [127.0.0.1]:5701 [dev] [5.1-SNAPSHOT] Shutdown request of Member [127.0.0.1]:5701 - beef212d-c629-49e3-908b-032fd4b75b0d this is handled
09:31:18,329  INFO |testUserCancel| - [Node] Time-limited test - [127.0.0.1]:5701 [dev] [5.1-SNAPSHOT] Shutting down connection manager...
09:31:18,329  INFO |testUserCancel| - [Node] Time-limited test - [127.0.0.1]:5701 [dev] [5.1-SNAPSHOT] Shutting down node engine...
09:31:18,367  INFO |testUserCancel| - [NodeExtension] Time-limited test - [127.0.0.1]:5701 [dev] [5.1-SNAPSHOT] Destroying node NodeExtension.
09:31:18,368  INFO |testUserCancel| - [Node] Time-limited test - [127.0.0.1]:5701 [dev] [5.1-SNAPSHOT] Hazelcast Shutdown is completed in 50 ms.
09:31:18,368  INFO |testUserCancel| - [LifecycleService] Time-limited test - [127.0.0.1]:5701 [dev] [5.1-SNAPSHOT] [127.0.0.1]:5701 is SHUTDOWN
BuildInfo right after testUserCancel(com.hazelcast.jet.sql.SqlErrorTest): BuildInfo{version='5.1-SNAPSHOT', build='20220110', buildNumber=20220110, revision=5e78d1d, enterprise=false, serializationVersion=1}
Hiccups measured while running test 'testUserCancel(com.hazelcast.jet.sql.SqlErrorTest):'
09:31:15, accumulated pauses: 912 ms, max pause: 105 ms, pauses over 1000 ms: 0


No metrics recorded during the test
```

</details>

--- a/hazelcast-sql/src/test/java/com/hazelcast/jet/sql/SqlErrorAbstractTest.java
+++ b/hazelcast-sql/src/test/java/com/hazelcast/jet/sql/SqlErrorAbstractTest.java
@@ -122,14 +122,10 @@ public abstract class SqlErrorAbstractTest extends SqlTestSupport {
         instance1 = newHazelcastInstance(true);
         client = newClient();
 
-        createMapping(instance1, MAP_NAME, long.class, long.class);
-        IMap<Long, Long> map = instance1.getMap(MAP_NAME);
-        map.put(1L, 1L);
-        map.put(2L, 2L);
-
         HazelcastInstance target = useClient ? client : instance1;
 
-        try (SqlResult res = target.getSql().execute(query().setCursorBufferSize(1))) {
+        try (SqlResult res = target.getSql().execute("select * from table(generate_stream(1))")) {
+            sleepSeconds(1);
             res.close();
 
             try {

